examples/chip-tool/commands/common/Utils.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_UTILS_H__
+#define __CHIPTOOL_UTILS_H__
+
+#include <memory>
+#include <vector>
+
+class Command;
```

Comment:
should we #include instead?

Reasoning being: these structures/methods would not be usable without a definision of Command anyway and #include would prevent us from creating some circular dependencies.

as an aside, I am somewhat concerned about anything named 'helpers' or 'utils' (not obvious what they are when #included, end up being a mixed bad of everything inside). Maybe we could make these something else (CommandBuilders, CommandLists or just Lists).

Reply:
I got your point about the name. I will rename it to something more specific.

Reply:
I removed the file and move the helpers directly into Commands.h.